pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -812,31 +814,41 @@ def show_help_instancename():
 """)
 
 
-def get_instancename(context, instancename, options, ns_names=None):
+def get_instancename(context, instancename, options, default_all_ns=False):
     """
     Common function to construct a CIMInstanceName object from the
     INSTANCENAME argument and the --key and --namespace options specified
     in the command line.
 
-    The keybindings of the returned CIM instance path must be specified in
+    If the ns_names option is a list, the request is to return instance
+    names in any of the namespaces.  Otherwise this function processes a
+    single namespace, the namespace defined by ns_names (string), the
+    namespace defined by the namespace component of the instancename or
+    the default namespace.
+
+    The keybindings of the returned CIM instance path(s) must be specified in
     exactly one of these ways:
 
     * If the keybindings component in the instance name string is "?"
       (e.g. "CIM_Foo.?"), the instances of that class are listed and the user is
-      prompted to pick one.
+      prompted to pick instance name. If multiple namespaces are specified
```

Comment:
On "If multiple namespaces are specified": With the new parameter, there is no list of namespaces specified anymore, so this phrase sounds a bit strange to me. Change to match the semantics of the new boolean parameter.